.github/workflows/release.yml

```diff
@@ -2,7 +2,7 @@ name: Release CI
 on:
   push:
     branches:
-      - master
+      - nau/nutmeg.master
```

Comment:
Change to nau/*.master so we don't need to change it on each distribution upgrade.